docs/providers/kubeless/cli-reference/deploy.md

```diff
@@ -0,0 +1,42 @@
+<!--
+title: Serverless Framework Commands - Kubeless - Deploy
+menuText: deploy
+menuOrder: 2
+description: Deploy your service to the specified provider
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/kubeless/cli-reference/deploy)
+<!-- DOCS-SITE-LINK:END -->
+
+# Kubeless - Deploy
+
+The `sls deploy` command deploys your entire service via the Kubeless API. Run this command when you have made service changes (i.e., you edited `serverless.yml`).  
+
+Use `serverless deploy function -f my-function` when you have made code changes and you want to quickly upload your updated code to your Kubernetes cluster or when you want to deploy just a function of your Serverless service.
```

Comment:
Right now `serverless deploy function` is just for updating. I opened an issue for deploying the function if it still doesn't exist: serverless/serverless-kubeless/issues/52